ibm_cloud_sdk_core/authenticators/iam_authenticator.py

```diff
@@ -56,18 +56,20 @@ class IAMAuthenticator(Authenticator):
 
     def __init__(self,
                  apikey: str,
+                 *,
                  url: Optional[str] = None,
                  client_id: Optional[str] = None,
                  client_secret: Optional[str] = None,
                  disable_ssl_verification: Optional[bool] = False,
```

Comment:
I don't think we want `Optional` here.

`Optional` should only be used when the default value is `None`.

https://docs.python.org/3/library/typing.html#typing.Optional